doc/stream.tex

```diff
@@ -306,9 +306,9 @@
 \end{procedure}
 \returns{} a transformer
 
-The \code{s/drop-while} procedure returns a transformer that returns the first tail that
-starts with the first value for which \var{procedure} returns false. If \var{procedure}
-returns a true value for all values, it returns the empty stream.
+The \code{s/drop-while} procedure returns a transformer that returns the tail starting
```

Comment:
Since it returns a stream, we could say that it returns the tail of the stream starting ...